.gitignore

```diff
@@ -53,3 +53,8 @@ local-running/bin/
 
 # User-specific secrets
 secrets.gradle
+
+api-tests/cucumber-report.html
+api-tests/.npmrc
+api-tests/.env
+api-tests/src/**/*.js
```

Comment:
(This addition looks a little suspicious...)

Reply:
I added this as I ran `npx tsc` to try compiling stuff with typecript directly, rather than using ts-node. And it generated all the js files, which we don't want. Is there a better way to deal with this?

Reply:
You can add `noEmit: true` to your tsconfig.json to prevent it from emitting JS files - then `tsc` will just do type-checking.